TypeCobol/Compiler/CodeElements/Expressions/StorageArea.cs

```diff
@@ -110,17 +110,17 @@ public enum StorageAreaKind
 	/// defined special registers (see list in a comment just below).
 	/// </summary>
 	public class DataOrConditionStorageArea: StorageArea {
```

Comment:
You did not change the Equals/GetHashCode for StorageArea.

Reply:
In StorageArea, there is no Equals/GetHashCode.

Reply:
Yes but that's the point. @smedilol asked for a custom equality comparison mechanism for both StorageArea and Node so the idea is to check whether it is useless or not and to decide which implementation to use.